tests/Helios.FsCheck.Tests/Buffers/BufferHelpers.cs

```diff
@@ -6,16 +6,6 @@ namespace Helios.FsCheck.Tests.Buffers
 {
     public static class BufferHelpers
     {
-        public static List<List<T>> ChunkOps<T>(List<T> source, int chunkSize)
```

Comment:
Refactored into standalone class - thought I'd need it for TcpSocketChannel tests, but ended up not using it.